CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # App Center SDK for React Native Change Log
 
+## Version 1.12.0
+
+### AppCenter
+
+* **[Fix]** Fix issue where the SDK source could not be built in a directory that contains escaped characters (applies to all modules).
```

Comment:
Please remove that fix as it applies when building apple sdk from source, not react native SDK.